src/middlewares/authorization.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+
+require('dotenv').config();
+
+export const auth = (req, res, next) => {
+  const token = req.header('auth-token');
+  if (!token) return res.status(401).send({ msg: 'Access Denied' });
+  try {
+    const privateKey = process.env.PRIVATE_KEY;
+    const validToken = jwt.verify(token, privateKey);
+    req.user = validToken;
+    return next();
+  } catch (error) {
+    return res.status(400).send({ msg: 'Invalid token' });
```

Comment:
The code should not be 400, it is not a bad request, it is an unauthorized request.